### PR TITLE
feat: mmdocs in playground results

### PR DESCRIPTION
**Pull request by florian-hoenicke**
Support multiple fields in the results

image

### CODE REVIEW

**Review by jupyterjazz — APPROVED** [review]
LGTM!

I just tried this out and the only small issue is that even if I specify only image to be my index field it still shows image + text in the playground. This might not seem an issue for this case but if we had several fields the results wouldn't be pretty. 

The issue will be fixed after we put non-index fields into tags. I'll open a ticket for this.

File: deployment/playground/playground.py [hunk 1]
```diff
@@ -415,44 +410,37 @@ def render_matches():
...
+    for chunk in match.chunks:
+        render_graphic_result(chunk, c)
+        render_text_result(chunk, c)
+
 
-    if match.uri != '':
+# I'm not so happy about these two functions, let's refactor them
+def render_graphic_result(match, c):
+    try:
+        match.mime_type = 'text-or-image-or-video'
```
[the remark this answers is not in the document]
Comment (florian-hoenicke) [reply deployment/playground/playground.py:422]: yes, otherwise it complains that no mimetype is set. This was there before

**Review by IyadhKhalfallah — APPROVED** [review]
LGTM! 🚀

**Comment by florian-hoenicke** [comment]
> LGTM!
> 
> I just tried this out and the only small issue is that even if I specify only image to be my index field it still shows image + text in the playground. This might not seem an issue for this case but if we had several fields the results wouldn't be pretty.
> 
> The issue will be fixed after we put non-index fields into tags. I'll open a ticket for this.

I think, we should show all the fields of the document even if fields are not used for indexing

**Pull request merged**